feature-libs/checkout/base/occ/adapters/occ-checkout-delivery-address.adapter.spec.ts

```diff
@@ -0,0 +1,160 @@
+import {
+  HttpClientTestingModule,
+  HttpTestingController,
+} from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { CheckoutState } from '@spartacus/checkout/base/root';
+import {
+  Address,
+  ADDRESS_NORMALIZER,
+  ADDRESS_SERIALIZER,
+  Cart,
+  ConverterService,
+  OccConfig,
+  OccEndpoints,
+} from '@spartacus/core';
+import { take } from 'rxjs/operators';
+import { OccCheckoutDeliveryAddressAdapter } from './occ-checkout-delivery-address.adapter';
+
+const checkoutData: Partial<CheckoutState> = {
+  deliveryAddress: {
+    firstName: 'Janusz',
```

Comment:
who's that! haha

Reply:
lol, no idea haha. The test already existed